playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -363,6 +363,18 @@
     - service_variant_config
     - deploy
 
+# generate any declared database cache
+# tables (will make in default db)
+- name: create database cache tables
+  django_manage:
+    app_path: "{{ edxapp_code_dir }}"
+    command: "lms createcachetable"
+    settings: "{{ EDXAPP_SETTINGS }}"
+    virtualenv: "{{ edxapp_venv_dir }}"
+  environment: "{{ edxapp_environment }}"
+  ignore_errors: true
+  run_once: true
```

Comment:
Why do you think we need to set `run_once` to `True`? It will run everytime we do a deployment, but only once during a deployment, is that what we want? Is the command safe to run twice?

```suggestion
```

Reply:
Well, there's just one database to affect, so there should be no harm in running twice, but no need to run it from every edxapp instance.